dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/service/SoftwareService.java

```diff
@@ -36,82 +38,84 @@
    * Constants.
    */
   private static final ObjectMapper MAPPER = new ObjectMapper();
+  private static final SoftwareProperties config = new SoftwareProperties();
```

Comment:
Could you explain how to implement this properly?  I've been attempting to use the examples in the other service methods, but I think I'm missing something.